playbooks/roles/ambari-config/tasks/main.yml

```diff
@@ -18,7 +18,7 @@
     backup: no
     state: present
     regexp: "^gpl.license.accepted.*"
-    line: "gpl.license.accepted={{ (accept_gpl|default(omit)|bool == true) | ternary('true', 'false') }}"
+    line: "gpl.license.accepted={{ accept_gpl|default(omit)|bool | ternary('true', 'false') }}"
```

Comment:
Why not simply :
```
gpl.license.accepted={{ accept_gpl }}
or
gpl.license.accepted={{ accept_gpl |default('true') }}
```

Reply:
it covers more edge cases, not sure what you're trying to solve here...